src/tweets.test.ts

```
@@ -368,6 +368,23 @@ test('sendTweet successfully sends a tweet', async () => {
+test('scraper can delete tweet', async () => {
+  const scraper = await getScraper();
+
+  const draftText = 'This Tweet will be deleted' + Date.now().toString();
+
+  const response = await scraper.sendTweet(draftText);
+
+  const result = await response.json();
+
+  expect(result).toBeDefined();
+
+  const tweetId = result?.data?.create_tweet?.tweet_results?.result?.rest_id
```

ESLint

[error] Insert `;`

(prettier/prettier)